code/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -377,10 +377,10 @@
    "source": [
     "[global]\n",
     "# input can either be 1 vcf genoFile, or a list of vcf genoFile.\n",
-    "parameter: genoFile = path\n",
+    "parameter: genoFile = paths\n",
     "\n",
     "# Workdir\n",
-    "parameter: cwd = path\n",
+    "parameter: cwd = paths\n",
```

Comment:
this should be `path` not `paths`

Reply:
if we expect it to use the same function as genotype_reformatting, then this need to be paths...

Reply:
Oh nvm i did see it is cwd, indeed this should be path, changed